### PR TITLE
Dockerizing application

### PR DESCRIPTION
**Pull request by MichaelOlatunji**
I  added a dockerfile and .dockerignore file to containerize the application
Will be adding shell scripts to automate things soon
Note that I have run built a docker image of this application on AWS ec2 instance to showing the Dockerfile contains no bugs

Michael Olatunji , @imyke From devOps team

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
-const express = require('express');
-const user = require('../controllers/user_controller');
-
+const express = require("express");
```
Comment (tonymj76) [inline routes/route.js:1]: use the eslint we add revert this change

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Pull latest node image from docker
+FROM node:latest
+
+# Maintainer Michael Olatunji @imyke ID: HNG-05739
```
Comment (tonymj76) [inline Dockerfile:4]: this is a group work no need to personalize it

Comment (tonymj76) [reply Dockerfile:4]: it will show on the contribution page

File: routes/route.js (hunk 1 above)
Comment (MichaelOlatunji) [reply routes/route.js]: oops! my branch was some coming behind

File: Dockerfile (hunk 2 above)
Comment (MichaelOlatunji) [reply Dockerfile]: Removed

File: routes/route.js (hunk 1 above)
Comment (MichaelOlatunji) [reply routes/route.js]: Note: I'm working with the master branch, I created my branch from the master

File: run_docker.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (tonymj76) [inline run_docker.sh:1]: nice work

File: routes/route.js (hunk 1 above)
Comment (MichaelOlatunji) [reply routes/route.js]: Checking the master branch again now, no eslint there
